sovtokenfees/sovtokenfees/test/test_prod_minting.py

```diff
@@ -0,0 +1,13 @@
+from sovtoken.constants import ADDRESS, AMOUNT
+
+
+PROD_ATOM_COUNT = 8000000000 * 100000
```

Comment:
It turned out that 1 token = 10^8 sovatoms, so can you please update the test (to check for 8*10^17)